src/test/groovy/com/github/jk1/license/ReportTaskCachingSpec.groovy

```diff
@@ -32,16 +32,16 @@ class ReportTaskCachingSpec extends Specification {
     File localBuildCacheDirectory
 
     def setup() {
-        buildFile = new File(testProjectDir, 'build.gradle')
         localBuildCacheDirectory = new File(testProjectDir, '.local-cache')
         localBuildCacheDirectory.mkdir()
-        new File(testProjectDir, 'settings.gradle') << """
-        buildCache {
-            local {
-                directory '${localBuildCacheDirectory.toURI()}'
+        buildFile = new File(testProjectDir, 'settings.gradle') << """
```

Comment:
```suggestion
        new File(testProjectDir, 'settings.gradle') << """
```

this is `settingsFile` really, not `buildFile`.

Reply:
noted - thanks

Reply:
Fixed